docker-compose.yml

```diff
@@ -53,7 +53,6 @@ networks:
   application:
     driver: "bridge"
     name: wp-services
-    external: true
```

Comment:
This needs to stay here, because if it isn't then I can't have other services connect to it. I can add a `make network` command if that's helpful.